zulip/zulip/examples/delete-topic

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+import zulip
+
+usage = """delete-topic <streams_id, topic_name>
+
+Example: delete-topic 1 "Denmark1"
+"""
+
+
+parser = zulip.add_default_arguments(argparse.ArgumentParser(usage=usage))
+parser.add_argument('streams_id', type=int)
```

Comment:
It's clear you haven't tested this, because this should be `stream_id`, not `streams_id`.

Reply:
Also, I think this example should support passing a `stream_name` instead of a `stream_id`; it should be able to detect based on whether what it got is a number, and if it is a string, it can look up that stream by name using the API and act appropriately.

Reply:
Is there automatic testing for this? or just manual, yeah sure will do that.